tools/rosgraph/test/test_roslogging.py

```diff
@@ -32,7 +32,10 @@
 
 import logging
 import os
-from StringIO import StringIO
+try:
+    from cStringIO import StringIO
```

Comment:
It looks like the import `from io import StringIO` works in python 2.7. The API's for `StringIO` in both look the same, but I didn't try this locally.

* `io` [2.7 Documentation](https://docs.python.org/2.7/library/io.html#io.StringIO)
* `cStringIO` [2.7 documentation](https://docs.python.org/2/library/stringio.html#cStringIO.StringIO)

Reply:
Nevermind, I failed to read this: http://wiki.ros.org/python_2_and_3_compatible_code#StringIO

Reply:
Please see the comment on the ROS wiki page why this approach is used.